front/src/styles/scss/applications/stdcmV2/_card.scss

```diff
@@ -0,0 +1,44 @@
+.stdcm-v2-card {
+    border-radius: 8px;
+    box-shadow: 0px 1px 0px rgba(255, 255, 255, 1) inset, 0px 4px 7px -3px rgba(24, 68, 239, 0.19), 0px 1px 3px rgba(49, 46, 43, 0.2);
+    background-color: rgba(255, 255, 255, 1);
+    height: fit-content;
+    position: relative;
+
+    &.has-tip::after {
+        content: "";
+        position: absolute;
+        top: 100%;
+        left: 20px;
+        border-top: 20px solid transparent;
+        border-top-color: rgba(255, 255, 255, 1); 
+        border-left: 20px solid transparent;
+        border-right: 20px solid transparent; 
```

Comment:
![Capture d’écran 2024-05-23 à 15 42 24](https://github.com/OpenRailAssociation/osrd/assets/113104137/67947526-a23b-4233-9316-06ebdfbff75d)

The arrow is longer and doesn't have the box shadow from the model : 

![Capture d’écran 2024-05-23 à 15 35 42](https://github.com/OpenRailAssociation/osrd/assets/113104137/bc817787-eafa-4271-bbd1-35530954180a)